src/views/studio/StudioLayout.tsx

```diff
@@ -91,25 +98,25 @@ const StudioLayout = () => {
                 <PrivateRoute
                   path={relativeRoutes.studio.signIn()}
                   element={<SignInView />}
-                  isAuth={!memberSet && accountSet}
+                  isAuth={!!memberships?.length && !memberSet}
```

Comment:
If we need this membership condition in each of those, let's put that in a variable